src/lambda/local/debugConfigurationProvider.ts

```diff
@@ -0,0 +1,265 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import {
+    join
+} from 'path'
+import {
+    CancellationToken,
+    DebugConfiguration,
+    DebugConfigurationProvider,
+    WorkspaceFolder
+} from 'vscode'
+import {
+    mkdirAsync,
+    readdirAsync,
+    statAsync,
+    writeFileAsync
+} from '../../shared/filesystem'
+import { fileExists, readFileAsString } from '../../shared/filesystemUtilities'
```

Comment:
Nit: Inconsistency here in import formatting, sometimes multiline (even for one item), sometimes all on one line.

Reply:
✔️